backend/msgpack/queue.go

```diff
@@ -160,16 +160,14 @@ type queueMetrics struct {
 	connWriteErrors   tally.Counter
 }
 
-func newQueueMetrics(s tally.Scope, instanceID string, queueSize int) queueMetrics {
+func newQueueMetrics(s tally.Scope, queueSize int) queueMetrics {
 	numBuckets := defaultQueueSizeNumBuckets
 	if queueSize < numBuckets {
 		numBuckets = queueSize
 	}
 	buckets := tally.MustMakeLinearValueBuckets(0, float64(queueSize/numBuckets), numBuckets)
 	return queueMetrics{
-		queueSize: s.Tagged(
-			map[string]string{"instance-id": instanceID},
-		).Histogram("queue-size", buckets),
+		queueSize: s.Histogram("queue-size", buckets),
```

Comment:
Would this cause all the old ids with `instance-id` tags to be fetched? Do we wanna change it to a different name perhaps?